LLMonFHIR.xcodeproj/project.pbxproj

```diff
@@ -664,7 +668,7 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = 637867499T;
+				DEVELOPMENT_TEAM = Y65AD5HVCG;
```

Comment:
The release configuration should stay the same using @PSchmiedmayer's development team.
<img width="1178" alt="Screenshot 2023-07-12 at 6 36 12 PM" src="https://github.com/StanfordBDHG/LLMonFHIR/assets/28656495/aac87fd2-ab57-43e4-a4b2-2f061336e868">